infra/gcb/builds_status.py

```diff
@@ -24,28 +42,32 @@
 BUILDS_PAGE_SIZE = 256
 BADGE_IMAGE_TYPES = {'svg': 'image/svg+xml', 'png': 'image/png'}
 
-_client = None
+_CLIENT = None
```

Comment:
this shouldn't be all caps, as it's not a constant. Did lint complain here? If so, let's add a ignore.

Reply:
Yeah lint complained, adding an ignore